added new labs

JBAnderson5 opened the pull request:
@abhinavoracle This isn't ready to merge yet, but I wanted to share what I have so far.

I'm adding two steps to the provisioning lab: Creating a live version of the app and Giving end users access to the app

I also started working on one of the two labs Dennis mentioned on our call today. Not sure exactly what to call these labs, so we can definitely change them if they don't make sense to you.

"try" lab or "Try out ATOM and learn about it's features"
This lab will go through 
1. Accessing the Application
2. Chat with LLM with base knowledge
3. Chat with LLM with augmented knowledge

"modify" lab I haven't started on yet, but this would contain instructions for how to modify the VB UI and the ODA skill to customize it or extend it to other use cases.

LMK your thoughts on this

JBAnderson5 committed:
added a couple new labs and started working on filling in the content for them

abhinavoracle commented:
> @abhinavoracle This isn't ready to merge yet, but I wanted to share what I have so far.
> 
> I'm adding two steps to the provisioning lab: Creating a live version of the app and Giving end users access to the app
> 
> I also started working on one of the two labs Dennis mentioned on our call today. Not sure exactly what to call these labs, so we can definitely change them if they don't make sense to you.
> 
> "try" lab or "Try out ATOM and learn about it's features" This lab will go through
> 
> 1. Accessing the Application
> 2. Chat with LLM with base knowledge
> 3. Chat with LLM with augmented knowledge
> 
> "modify" lab I haven't started on yet, but this would contain instructions for how to modify the VB UI and the ODA skill to customize it or extend it to other use cases.
> 
> LMK your thoughts on this

Sounds good JB! Let me know if you need any help from my side.

JBAnderson5 committed:
fixed merge conflicts

JBAnderson5 committed:
added new workshop for generative-ai-agent-oda

JBAnderson5 committed:
started work on outlining the labs

JBAnderson5 committed:
Merge branch 'main' of github.com:abhinavoracle/analytics-ai

pulling down changes from upstream

JBAnderson5 committed:
moved content from word to git for intro and agent lab. updated manifests

JBAnderson5 committed:
updated terraform lab

JBAnderson5 committed:
created functions lab

JBAnderson5 committed:
moved oda and vb labs from word to markdown

kaushik-kundu committed:
Add files via upload

kaushik-kundu committed:
Update introduction.md

kaushik-kundu committed:
Add files via upload

kaushik-kundu committed:
Update agent.md

kaushik-kundu committed:
Add files via upload

kaushik-kundu committed:
Update agent.md

kaushik-kundu committed:
Add files via upload

kaushik-kundu committed:
Update agent.md

JBAnderson5 committed:
updated genai agent oda lab. updated screenshots and linked to hosted artifacts

JBAnderson5 committed:
updated manifest files and added link to atom livelab

JBAnderson5 committed:
removed learn more section from all but intro lab

JBAnderson5 committed:
lots of minor fixes.

JBAnderson5 committed:
added instructions to view ODA analytics and add users to VB app

JBAnderson5 committed:
updated vb instructions for promoting app to live and adding users to app

JBAnderson5 committed:
added optional instructions to customize ODA chat

JBAnderson5 committed:
clarified oda instructions

JBAnderson5 committed:
merging in upstream changes

abhinavoracle committed:
WMS ID : 11617 Updating few instructions in the labs (#329)

* Added ATOM steps

* Update manifest.json

* Updated images

* update manifest.json

* Update manifest.json

* Added zip files

* Updated few links

* Update introduction.md

* Updated files

* Updated files

* updated intro

* Update provision.md

* Update introduction.md

* Updated files

* completed first draft of intro

* Updated files

* Updated files

* Updated files

* Updated files

* Update provision.md

* Updated need help links

* Update provision.md

* Update provision.md

* Deleted images from sample workshop

* Updated Provision.md

* Updated Provision.md

* Updated Provision.md and addded and image

* Update provision.md

* Update provision.md

* Update provision.md

* Updated files

* Added another lab

Added another lab

* Update provision.md

Updated few steps

* Updated names

* Updated

Updated

* Updated

* Updated files

* Updated file

Updated

* Updated instructions

Updated instructions

* Updated files

Updated files

* Updated files

Updated files

* Updated date

---------

Co-authored-by: JB Anderson <jb.anderson@oracle.com>

MarcGueury committed:
WMS 11670 - Hybrid Search with PostgresSQL (#330)

* ocw feedback

* genai1

* Update manifest.json

* GenAI + Opensearch 2.11

* GENAI

* Known issues

* Update integration.md

* Several fixes mostly for multi region.

* a

* Create genai_agent.md

* Create images folder

* Add files via upload

* Update genai_agent.md

* added Lab 4 (Optional): GenAI Agent

* Update genai_agent.md

* Add files via upload

* Delete a.txt

* completed genai_agent.md

* added genai agent screenshots

* formating changes genai_agent.md

* postgres v1

* v2

* initial  discover.md

* Delete oci-postgresql-genai-lab/3-discover/images directory

* Create images folder

* added screenshots for lab3

* Update discover.md

* PostgresLab

* Update test.md

* -

* Hybrid

* qa

* Create .gitignore

* remove genai agent

* hybrid

* screenshot

---------

Co-authored-by: mkratky <marek.kraatkyy@gmail.com>
Co-authored-by: Hope Fisher <127253314+hope-fisher@users.noreply.github.com>

JBAnderson5 committed:
worked through some TODOs and started user lab

JBAnderson5 committed:
minor formatting fixes

JBAnderson5 committed:
trying copy codeblock without specified language

JBAnderson5 committed:
trying new codeblock syntax

JBAnderson5 committed:
updated all codeblocks to be copyable

JBAnderson5 committed:
updating instructions

JBAnderson5 committed:
lots of minor tweaks going through qa checklist

kaushik-kundu committed:
Update oda.md

MarcGueury committed:
WMS 11670 - Hybrid Search with PostgresSQL - Change in wording asked by PM (#331)

* ocw feedback

* genai1

* Update manifest.json

* GenAI + Opensearch 2.11

* GENAI

* Known issues

* Update integration.md

* Several fixes mostly for multi region.

* a

* Create genai_agent.md

* Create images folder

* Add files via upload

* Update genai_agent.md

* added Lab 4 (Optional): GenAI Agent

* Update genai_agent.md

* Add files via upload

* Delete a.txt

* completed genai_agent.md

* added genai agent screenshots

* formating changes genai_agent.md

* postgres v1

* v2

* initial  discover.md

* Delete oci-postgresql-genai-lab/3-discover/images directory

* Create images folder

* added screenshots for lab3

* Update discover.md

* PostgresLab

* Update test.md

* -

* Hybrid

* qa

* Create .gitignore

* remove genai agent

* hybrid

* screenshot

* pgvector

* -

---------

Co-authored-by: mkratky <marek.kraatkyy@gmail.com>
Co-authored-by: Hope Fisher <127253314+hope-fisher@users.noreply.github.com>

JBAnderson5 committed:
updated vb lab

JBAnderson5 committed:
updated par links

JBAnderson5 committed:
pulling in upstream changes

JBAnderson5 committed:
updated syntax for oda lab
